bigquery/pom.xml

```diff
@@ -0,0 +1,197 @@
+  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <artifactId>zeppelin</artifactId>
+    <groupId>org.apache.zeppelin</groupId>
+    <version>0.7.0-SNAPSHOT</version>
+  </parent>
+
+  <groupId>org.apache.zeppelin</groupId>
+  <artifactId>zeppelin-bigquery</artifactId>
+  <packaging>jar</packaging>
+  <version>0.7.0-SNAPSHOT</version>
+  <name>Zeppelin: BigQuery interpreter</name>
+  <url>http://www.apache.org</url>
+
+  <dependencies>
+
+    <dependency>
+        <groupId>com.google.apis</groupId>
+        <artifactId>google-api-services-bigquery</artifactId>
+        <version>v2-rev265-1.21.0</version>
+    </dependency>
+    <dependency>
+        <groupId>com.google.oauth-client</groupId>
+        <artifactId>google-oauth-client</artifactId>
+        <version>${project.oauth.version}</version>
+    </dependency>
+    <dependency>
+        <groupId>com.google.http-client</groupId>
+        <artifactId>google-http-client-jackson2</artifactId>
+        <version>${project.http.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>com.google.oauth-client</groupId>
+      <artifactId>google-oauth-client-jetty</artifactId>
+      <version>${project.oauth.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>com.google.code.gson</groupId>
+      <artifactId>gson</artifactId>
+      <version>2.6</version>
+    </dependency>
+
+    <dependency>
+      <groupId>org.apache.zeppelin</groupId>
+      <artifactId>zeppelin-interpreter</artifactId>
+      <version>${project.version}</version>
+      <scope>provided</scope>
+    </dependency>
+
+    <dependency>
+      <groupId>org.slf4j</groupId>
+      <artifactId>slf4j-api</artifactId>
+    </dependency>
+
+    <dependency>
+      <groupId>org.slf4j</groupId>
+      <artifactId>slf4j-log4j12</artifactId>
+    </dependency>
+
+
+    <dependency>
+      <groupId>com.google.guava</groupId>
+      <artifactId>guava</artifactId>
+    </dependency>
+
+    <dependency>
+      <groupId>junit</groupId>
+      <artifactId>junit</artifactId>
+      <scope>test</scope>
+    </dependency>
+
+    <dependency>
+      <groupId>org.mockito</groupId>
+      <artifactId>mockito-all</artifactId>
+      <version>1.9.5</version>
+      <scope>test</scope>
+    </dependency>
+
+    <dependency>
+      <groupId>com.mockrunner</groupId>
+      <artifactId>mockrunner-jdbc</artifactId>
+      <version>1.0.8</version>
+      <scope>test</scope>
+    </dependency>
+  </dependencies>
+
+  <properties>
+    <project.http.version>1.21.0</project.http.version>
+    <project.oauth.version>1.21.0</project.oauth.version>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <bigquery.test.exclude>**/BigQueryInterpreterTest.java</bigquery.test.exclude>
```

Comment:
Excluding tests will fix the CI, but for the future maintainers, assuming they will not know much about it at first, I think we need to have few things documented in `./bigquery/README.md` 
 - a command for manually running it `mvn -Dbigquery.test.exclude='' test -pl bigquery`
 - a pre-requests, that need to be configured (and how) in order for this test to pass locally 

I tried on my local env and got

```
$mvn -Dbigquery.test.exclude='' test -pl bigquery
....
java.lang.NullPointerException: null
	at org.apache.zeppelin.bigquery.BigQueryInterpreter.run(BigQueryInterpreter.java:261)
	at org.apache.zeppelin.bigquery.BigQueryInterpreter.executeSql(BigQueryInterpreter.java:246)
	at org.apache.zeppelin.bigquery.BigQueryInterpreter.interpret(BigQueryInterpreter.java:287)
	at org.apache.zeppelin.bigquery.BigQueryInterpreterTest.badSqlSyntaxFails(BigQueryInterpreterTest.java:110)

sqlSuccess(org.apache.zeppelin.bigquery.BigQueryInterpreterTest)  Time elapsed: 0 sec  <<< ERROR!
java.lang.NullPointerException: null
	at org.apache.zeppelin.bigquery.BigQueryInterpreter.run(BigQueryInterpreter.java:261)
	at org.apache.zeppelin.bigquery.BigQueryInterpreter.executeSql(BigQueryInterpreter.java:246)
	at org.apache.zeppelin.bigquery.BigQueryInterpreter.interpret(BigQueryInterpreter.java:287)
	at org.apache.zeppelin.bigquery.BigQueryInterpreterTest.sqlSuccess(BigQueryInterpreterTest.java:101)

Results :

Tests in error:
  BigQueryInterpreterTest.badSqlSyntaxFails:110 » NullPointer
  BigQueryInterpreterTest.sqlSuccess:101 » NullPointer

Tests run: 2, Failures: 0, Errors: 2, Skipped: 0

[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
```

which leaves not much clues on what went wrong. What do you think, does it make sence?